src/middlewared/middlewared/main.py

```diff
@@ -550,12 +551,11 @@ def run(self):
         self.shell_pid, master_fd = os.forkpty()
         if self.shell_pid == 0:
             for i in range(3, 1024):
-                if i == master_fd:
-                    continue
```

Comment:
Why this change?

Reply:
It was useless, in child process `master_fd` is `-1`

Reply:
In that case we should use bsd.closefrom, which is much better than iterating 1021 times